src/service/platform-impl.js

```diff
@@ -94,7 +104,8 @@ export class Platform {
    * @return {boolean}
    */
   isWebKit() {
-    return /WebKit/i.test(this.navigator_.userAgent) && !this.isEdge();
+    return /WebKit/i.test(this.navigator_.userAgent) && !this.isEdge()
```

Comment:
hmm, Opera is Webkit, any particular reason this was added?

Reply:
Oh, I'll delete this.